packages/mvsp/mvsp-alpha.en.asciidoc

```diff
@@ -160,12 +161,11 @@ h| Description
 
 | 4.3 Subprocessors
 | * Maintain a list of third-party companies with access to customer data, and make it available to clients and business partners upon request
-  * Assess third-party companies annually against this baseline
+  * Assess third-party companies annually against the latest MVSP checklist
```

Comment:
Same as above: "latest MVSP release" or "latest stable MVSP checklist" or similar would be less confusing.